Misc/NEWS.d/next/Tests/2019-10-05-15-50-24.bpo-38377.gKfVDt.rst

```diff
@@ -0,0 +1,5 @@
+Add the environment variable :data:`PYTHON_NO_DEV_SHM`,
+which when set to 1, makes the Python skip tests depending
+on :data:`/dev/shm` interface, which are missing in building
+environments of some Linux distributions. Currently skipping
```

Comment:
IMHO we cannot support something that is not used for linux "standard"?

I am not an expert, but this sound like a specific distribution issue, and will be a little difficult (maybe)
pay attention for all Linux distributions issues.

Reply:
I don’t understand what’s your point. Do you think we should care less about Linux distributions, or what?

Reply:
Hmm I think so, I saw some other projects that when the issues come from an specific distribution, that can be fixed simply with a patch on the distribution package, and don't try to skip (in this case the test) on the upstream. It's just an opinion :)

Reply:
OK, this is just an opinion and not an action item.